lib/promscrape/config.go

```diff
@@ -37,7 +37,7 @@ var (
 // Config represents essential parts from Prometheus config defined at https://prometheus.io/docs/prometheus/latest/configuration/configuration/
 type Config struct {
 	Global        GlobalConfig   `yaml:"global"`
-	ScrapeConfigs []ScrapeConfig `yaml:"scrape_configs"`
+	ScrapeConfigs []ScrapeConfig `yaml:"scrape_configs,omitempty"`
```

Comment:
I'm unsure whether `Config` is useful without `scrape_configs` list

Reply:
you're right